resources/js/orderjs.js

```diff
@@ -80,3 +80,81 @@ $(document).on("click", ".order", function () {
         }
     });
 });
+
+$(document).ready(function (){
+    let url = $(".detail_order").data('url');
+    if (url) {
+        $.ajax({
+            url : url,
+            type : "GET",
+            success : function (data) {
+                $(".loader").hide();
+                $("#detail-Order").html(data);
+                $("#detail-Order").modal("show");
+            },
+            error : function (data) {
+                $(".loader").hide();
+                alert("Product not exists");
+            },
+            complete : function () {
+                $(".approved-order").click(function () {
+                    let url = this.getAttribute('data-url');
+                    $.ajax({
+                        url : url,
+                        type: "PATCH",
+                        headers: {
+                            'X-CSRF-TOKEN': $('meta[name="csrf-token"]').attr('content')
+                        },
+                        success : function (response) {
+                            let data = JSON.parse(response);
+                            alert(data.message);
+                            $(".status-order div").each(function (){
+                                if ($(this).data('id') == data.id) {
+                                    $(this).removeClass();
+                                    $(this).addClass("alert alert-info");
+                                    $(this).text(data.approved);
+                                }
+                            });
+                            $("#detail-Order").modal('hide');
```

Comment:
```suggestion
                            $("#detail-order").modal('hide');
```
sửa lại chỗ tên này đi em, cẩn thân vào em